src/Classes/Particle/Updater.ts

```diff
@@ -45,13 +45,32 @@ export class Updater {
         const particle = this.particle;
 
         if (options.particles.move.enable) {
-            const moveSpeed = options.particles.move.speed / 2;
+            let moveSpeed = options.particles.move.speed / 2;
+            moveSpeed *= this.getProximitySpeedFactor();
```

Comment:
@matteobruni  In this later commit I've added this method to calculate the speed factor based on mouse proximity as well as the 'slow' options

Reply:
Thanks, it's perfect now!

Reply:
So glad I could help!